src/allmydata/test/cli/wormholetesting.py

```diff
@@ -61,23 +60,23 @@ class MemoryWormholeServer(object):
     :ivar _waiters: Observers waiting for a wormhole to be created for a
         specific application id and relay URL combination.
     """
-    _apps: dict[ApplicationKey, _WormholeApp] = field(default=Factory(dict))
-    _waiters: dict[ApplicationKey, Deferred] = field(default=Factory(dict))
+    _apps: Dict[ApplicationKey, _WormholeApp] = field(default=Factory(dict))
+    _waiters: Dict[ApplicationKey, Deferred[Any]] = field(default=Factory(dict))
 
     def create(
         self,
-        appid,
-        relay_url,
-        reactor,
-        versions={},
-        delegate=None,
-        journal=None,
-        tor=None,
-        timing=None,
-        stderr=stderr,
-        _eventual_queue=None,
-        _enable_dilate=False,
-    ):
+        appid: Any,
+        relay_url: Any,
+        reactor: Any,
```

Comment:
For reactor you can use one of the interfaces in `twisted.internet.interfaces`, there's a variety (not sure if there's a catch-all interface, there might be.)

Reply:
Thanks for pointing me in the right direction. Looked into it there does not seem like there is an all encompassing reactor interface.